src/index.html

```diff
@@ -11,7 +11,98 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname:
```

Comment:
```suggestion
        <label class="form__field">
          Surname:
```
put text in label on a new line